### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by bhluong1codes**
Capture

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <label for="dropdown">What is coding cool?</label>
+      </div>
+      <select id="dropdown" name="dropdown">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        <label for="language">Favorite Language (choose one)</label>
```
Comment (Zhiwutian) [inline html-forms/index.html:23]: instead of a label here, I would put the text directly in a div, and wrap a label around each of the inputs.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
very good! Just a small label update for the radio buttons

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <label><input type="radio" name="language" value="css">CSS</label>
+        <label><input type="radio" name="language" value="js">Javascript</label>
+      <div>
+        <label for="signup">Would you like to sign up for more information?</label>
+      </div>
+        <input type="checkbox" name="signup" value="signup" id="signup">Sign me up!
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="comments"></textarea>
```
Comment (Zhiwutian) [inline html-forms/index.html:33]: make sure to add the id attribute here to match the label above

**Review by Zhiwutian — APPROVED** [review]
Approved! But make sure to update the id on the textarea. Sorry I missed that on the last check!

File: html-forms/index.html (hunk 2 above)
Comment (bhluong1codes) [reply html-forms/index.html]: Thanks for the feedback!

**Pull request merged**